release/release_after_kokoro.sh

```diff
@@ -0,0 +1,195 @@
+#!/bin/sh
+
+# This script starts with Step 14 in the release process (i.e., after Kokoro
+# has built and signed the binaries.)
+#
+# It is safe to re-run this script again when something goes wrong, provided
+# that you didn't complete the final step, which means you are done. (The final
+# step grants public access to files uploaded to cloud-tools-for-eclipse.)
+#
+# Note: this script clears and creates "$HOME/CT4E_release_work" (after
+# confirmation) for temporary work directory, which can be inspected later
+# and/or deleted safely anytime.
+
+WORK_DIR=$HOME/CT4E_release_work
+SIGNED_DIR=$WORK_DIR/signed
+LOCAL_REPO=$WORK_DIR/repository
+
+ask_proceed() {
+    while true; do
+        echo -n "Proceed ([Y]/n)? "
+        read ANSWER
+        [ "$ANSWER" == 'n' -o "$ANSWER" == 'N' ] && exit 1
+        [ "$ANSWER" == 'y' -o "$ANSWER" == 'Y' -o -z "$ANSWER" ] && break
+        echo -n "Invalid answer. "
+    done
+}
+
+###############################################################################
+echo "#"
+echo "# Clear '$WORK_DIR'."
+echo "#"
+ask_proceed
+
+set -x
+rm -rf $WORK_DIR && mkdir $WORK_DIR
+set +x
+
+###############################################################################
+echo
+echo "#"
+echo "# Copy files built (signed) by Kokoro"
+echo "# into '$SIGNED_DIR'."
+echo "#"
+
+echo -n "Enter GCS bucket URL taken from the Kokoro page: "
+read GCS_URL
+GCS_BUCKET_URL=$( echo $GCS_URL | sed -e 's,.*\(signedjars/staging/prod/google-cloud-eclipse/ubuntu/release/[0-9]\+/[0-9-]\+\),\1,' )
+echo "GCS URL extracted: $GCS_BUCKET_URL"
+echo -n "Check if the URL is correct. "
+ask_proceed
+
+set -x
+mkdir $SIGNED_DIR && \
+    gsutil -m cp -R gs://$GCS_BUCKET_URL/gfile/signed/* $SIGNED_DIR
+set +x
+
+if [ 0 -eq $( ls -1 $SIGNED_DIR | wc -l ) ]; then
+    echo "'$SIGNED_DIR' is empty. Wrong GCS bucket URL provided?"
+    echo "Fix the problem and rerun the script. Halting."
+    exit 1
+fi
+
+###############################################################################
+echo
+echo "#"
+echo "# Verify if constants have been injected..."
+echo "#"
+ask_proceed
+
+set -x
+javap -private -classpath $SIGNED_DIR/plugins/com.google.cloud.tools.eclipse.appengine.login_0.1.0.*.jar \
+       -constants com.google.cloud.tools.eclipse.appengine.login.Constants
+javap -classpath $SIGNED_DIR/plugins/com.google.cloud.tools.eclipse.usagetracker_0.1.0.*.jar \
+       -constants com.google.cloud.tools.eclipse.usagetracker.Constants
+set +x
+
+echo "Check the output above. "
+ask_proceed
+
+###############################################################################
+echo
+echo "#"
+echo "# Run Eclipse from the command line to generate the following files:"
+echo "#"
+echo "#     $LOCAL_REPO/artifacts.xml"
+echo "#     $LOCAL_REPO/content.xml"
+echo "#     $LOCAL_REPO/features/*"
+echo "#     $LOCAL_REPO/plugins/*"
+echo "#"
+ask_proceed
+
+echo "The following steps require running an Eclipse binary."
+while true; do
```

Comment:
I'm tempted to suggest that ECLIPSE_HOME should be set before running the script. Maybe if not set, default to the path on Goobuntu.